File: app/src/androidTest/java/com/inelasticcollision/recipelink/data/local/migration/helper/TestVersion2RecipeTable.java
package com.inelasticcollision.recipelink.data.local.migration.helper;

import android.provider.BaseColumns;

final class TestVersion2RecipeTable implements BaseColumns {

    // Table name
    static final String TABLE_NAME = "recipe";

    // Columns
    static final String COLUMN_DATE_ADDED = "date_added";
    static final String COLUMN_DATE_UPDATED = "date_updated";
    static final String COLUMN_TITLE = "title";
    static final String COLUMN_URL = "link";
    static final String COLUMN_IMAGE_URL = "photo_url";
    static final String COLUMN_FAVORITE = "favorite";
    static final String COLUMN_CATEGORY = "category";
    static final String COLUMN_KEYWORDS = "keywords";
    static final String COLUMN_NOTES = "notes";

}

File: app/src/androidTest/java/com/inelasticcollision/recipelink/data/cache/database/helper/TestVersion2RecipeTable.kt
package com.inelasticcollision.recipelink.data.cache.database.helper

import android.provider.BaseColumns

internal object TestVersion2RecipeTable : BaseColumns {
    // Table name
    const val TABLE_NAME = "recipe"

    // Columns
    const val COLUMN_DATE_ADDED = "date_added"
    const val COLUMN_DATE_UPDATED = "date_updated"
    const val COLUMN_TITLE = "title"
    const val COLUMN_URL = "link"
    const val COLUMN_IMAGE_URL = "photo_url"
    const val COLUMN_FAVORITE = "favorite"
    const val COLUMN_CATEGORY = "category"
    const val COLUMN_KEYWORDS = "keywords"
    const val COLUMN_NOTES = "notes"
}